src/style.css

```
@@ -0,0 +1,123 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.wrap {
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  /* flex-wrap: wrap; */
+  font-family: Arial, Helvetica, sans-serif;
+  color: white;
+  font-size: 100px;
+}
+
+.block {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 300px;
+  min-width: 300px;
+}
+
+.block--1 {
+  /* stylelint-disable-next-line color-hex-length */
+  background-color: #ff0000;
```

don't disable eslint rules, if you have hex value like this, you should use shorthand
```suggestion
  /* stylelint-disable-next-line color-hex-length */
  background-color: #f00;
```